# TODO Вызовите ошибку AssertionError с собственным текстом
raise AssertionError("Mistake")